android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlService.java

```diff
@@ -450,4 +531,97 @@ public void onError(String error) {
 			sdlManager.getScreenManager().presentChoiceSet(choiceSet, InteractionMode.MANUAL_ONLY);
 		}
 	}
+
+	public static class UIStreamingDisplay extends SdlRemoteDisplay {
+		int clickCounter1 = 0;
+		int clickCounter2 = 0;
+		public UIStreamingDisplay(Context context, Display display) {
+			super(context, display);
+		}
+
+		@Override
+		protected void onCreate(Bundle savedInstanceState) {
+			super.onCreate(savedInstanceState);
+			setContentView(R.layout.ui_streaming_layout);
+
+
+			final Button button1 = findViewById(R.id.button1);
+			final Button button2 = findViewById(R.id.button2);
+			final TextView counter1 = findViewById(R.id.button_one_counter);
+			final TextView counter2 = findViewById(R.id.button_two_counter);
+            button1.setOnTouchListener(new View.OnTouchListener() {
+				@Override
+				public boolean onTouch(View view, MotionEvent motionEvent) {
+
+                    counter1.setText("Click!!! " + ++clickCounter1);
+					Log.d("MyTagClickCounter", String.valueOf(clickCounter1));
+					int location [] = new int[2];
+                    button1.getLocationInWindow(location);
+                    //counter1.append("\nButton size: " + button1.getWidth() + "x" + button1.getHeight());
+					//textView.append("\nButton location: " + location[0] + "," + location[1]);
+					return false;
+				}
+			});
+
+            button2.setOnTouchListener(new View.OnTouchListener() {
+                @Override
+                public boolean onTouch(View view, MotionEvent motionEvent) {
+
+                    counter2.setText("Click!!! " + ++clickCounter2);
+                    Log.d("MyTagClickCounter", String.valueOf(clickCounter2));
+                    int location [] = new int[2];
+                    button2.getLocationInWindow(location);
+                    //counter2.append("\nButton size: " + button.getWidth() + "x" + button.getHeight());
+                    //textView.append("\nButton location: " + location[0] + "," + location[1]);
+                    return false;
+                }
+            });
+		}
+
+		@Override
+		public void onViewResized(int width, int height) {
+			Toast.makeText(getContext(),
+					String.format("Remote view new width and height (%s, %s)", width, height),
+					Toast.LENGTH_SHORT
+			).show();
+		}
+	}
+
+	public static class MyDisplay extends SdlRemoteDisplay {
```

Comment:
@santhanamk this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?